app/frontend/courses/review/components/CoursesReview__ChecklistShow.res

```diff
@@ -172,143 +173,174 @@ let make = (
         </button>
       </div>
       <div className="border bg-white rounded-lg py-2 md:py-4 mt-2 space-y-4">
-        {Js.Array.mapi((reviewChecklistItem, itemIndex) =>
-          <Spread
-            props={"data-checklist-item": string_of_int(itemIndex)} key={string_of_int(itemIndex)}>
-            <div>
-              <h4 className="relative text-sm font-semibold mt-2 md:mt-0 px-6 w-full md:w-4/5">
-                <div className={checklistItemCheckedClasses(itemIndex, selection)} />
-                {ReviewChecklistItem.title(reviewChecklistItem)->str}
-              </h4>
-              <div className="space-y-3 pt-3"> {Js.Array.mapi((checklistItem, resultIndex) =>
```

Comment:
@harigopal 

I think the name of the variable `checklistItem` should be replaced with `resultlistItem` as we have result array for each checklistitem.


```res
type t = {
  title: string,
  result: array<CoursesReview__ReviewChecklistResult.t>,
}
```
 
 what do you think ?